### ISSUE TITLE
a11y:  Try it out Response table is not accessible

### ISSUE THREAD

**Issue by cjfreedman** [swagger-api/swagger-ui#9574]
### Q&A (please complete the following information)
 - OS: [e.g. macOS]  Windows 10
 - Browser: [e.g. chrome, safari]  Chrome 
 - Version: [e.g. 22]   131
 - Method of installation: [e.g. npm, dist assets]  npm
 - Swagger-UI version: [e.g. 3.10.0]  swagger-ui-react 5.11.3
 - Swagger/OpenAPI version: [e.g. Swagger 2.0, OpenAPI 3.0]  OpenAPI 3.0

### Content & configuration
Basic default configuration

### Describe the bug you're encountering
We use a tool called Sort Site to evaluate our site for 508 accessibility compliance.  It is flagging that the Responses table is not accessible.

### To reproduce...
For a POST command, click Try it out and see the Responses table.

### Expected behavior
No accessibility error :)

### Screenshots
Here's the area of the screen
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/692a563f-c829-45fc-9355-f4832d0d58e5)

### Additional context or thoughts
Here's the Sort Site error info and solution
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/92e0d241-c983-4733-9b6e-06208ff7875d)
